doc/strace.1.in

```diff
@@ -1010,6 +1011,15 @@ Print the syscall number.
 .if '@USE_LIBDW_FALSE@'#' Print the execution stack trace and source code information of the traced
 .if '@USE_LIBDW_FALSE@'#' processes after each system call. This option expects the target program is compiled
 .if '@USE_LIBDW_FALSE@'#' with appropriate debug options: "\-g" (gcc), or "\-g \-gdwarf-aranges" (clang).
+.if '@ENABLE_STACKTRACE_FALSE@'#' .TP
+.if '@ENABLE_STACKTRACE_FALSE@'#' .B \-\-stacktrace\-frames\-limit [= limit ]
```

Comment:
The limit parameter is not optional. So you don't need `[` and `]` here.

Reply:
Done